pkg/rpc/client/client.go

```diff
@@ -25,9 +24,19 @@ var SuperSet = wire.NewSet(
 // RemoteURL for RPC remote host
 type RemoteURL string
 
+// Insecure for RPC remote host
+type Insecure bool
+
 // NewProtobufClient is the factory method to return RPC scanner
-func NewProtobufClient(remoteURL RemoteURL) rpc.Scanner {
-	return rpc.NewScannerProtobufClient(string(remoteURL), &http.Client{})
+func NewProtobufClient(remoteURL RemoteURL, insecure Insecure) (rpc.Scanner, error) {
```

Comment:
the same as the cache client

Reply:
Good catch. Thanks.

That's a left over from some other implementation I tried.